tests/clickhouse-test

```diff
@@ -3108,13 +3121,15 @@ def main(args):
             print(colored("\nNo queries hung.", args, "green", attrs=["bold"]))
 
     if args.client_log:
-        if os.path.exists(args.client_log):
-            with open(args.client_log, "rb") as stream:
+        for log_file in [args.client_log, *glob.glob(f"{args.client_log}.*")]:
+            if not os.path.exists(log_file):
+                continue
```

Comment:
Why do we need this check? I guess `args.client_log` always exists, as we didn't have a check before. That means `glob.glob` could return paths that doesn't exists? Or do I miss something here?

Reply:
We currently have `if os.path.exists(args.client_log):` in the code, so basically check is for that (`args.client_log` contains only fatal logs from client or doesn't exits). `glob` should return existing files, just in case I kept the check inside the loop

Reply:
Oh, I missed that, sorry. You are right.